qiskit_experiments/library/characterization/analysis/cr_hamiltonian_analysis.py

```diff
@@ -89,33 +86,18 @@ class CrossResonanceHamiltonianAnalysis(curve.CurveAnalysis):
                 .parametric_pulses.GaussianSquare` pulse envelope.
             bounds: [0, None]
 
-        defpar p_{x, 0}:
-            desc: Fit parameter of oscillations when control qubit state is 0.
+        defpar p_x:
+            desc: Fit parameter of oscillations.
```

Comment:
```suggestion
            desc: Fit parameter of oscillations of the X observable.
```
Same for Y and Z below.